flink-runtime/src/main/java/org/apache/flink/runtime/state/StateBackend.java

```diff
@@ -94,90 +95,27 @@ default String getName() {
      *
      * <p><i>Keyed State</i> is state where each value is bound to a key.
      *
-     * @param env The environment of the task.
-     * @param jobID The ID of the job that the task belongs to.
-     * @param operatorIdentifier The identifier text of the operator.
-     * @param keySerializer The key-serializer for the operator.
-     * @param numberOfKeyGroups The number of key-groups aka max parallelism.
-     * @param keyGroupRange Range of key-groups for which the to-be-created backend is responsible.
-     * @param kvStateRegistry KvStateRegistry helper for this task.
-     * @param ttlTimeProvider Provider for TTL logic to judge about state expiration.
-     * @param metricGroup The parent metric group for all state backend metrics.
-     * @param stateHandles The state handles for restore.
-     * @param cancelStreamRegistry The registry to which created closeable objects will be
-     *     registered during restore.
+     * @param parameters accessor.
```

Comment:
The doc for this argument is "accessor" everywhere, what does that even mean? Why not "argument bundle for creating ...." or something like that